packages/reaction-shipping/server/methods.js

```diff
@@ -74,9 +81,10 @@ Meteor.methods({
   updateShippingProvider: function (updateDoc, currentDoc) {
     check(updateDoc, Object);
     check(currentDoc, String);
-    if (!Roles.userIsInRole(Meteor.userId(), ['admin', 'shipping'])) {
```

Comment:
These are all legitimate replacements. There may have been a reason we're using 'userInRole' here at one time, but if this is working then this is our normal way of doing these validations.  The logic in return false instead of "Access Denied" is more just security by obscurity - the less info we give for your failure the better was probably the thinking.  Again, these updates are acceptable, but we also need to make sure the client can handle these responses as well.